### PR TITLE
refactor(upgrade ingress): add a flag to skip the update of jx related resources

### PR DESCRIPTION
**Pull request by ccojocar**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description

Add a flag to skip the update of jx related resource in the 'upgrade ingress' command. This allows to run the command 
before installing the jx platform. For instance when exposing the Vault used to stored the credentials.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: pkg/jx/cmd/upgrade_ingress.go [hunk 1]
```diff
@@ -89,36 +91,38 @@ func (o *UpgradeIngressOptions) addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringArrayVarP(&o.Namespaces, "namespaces", "", []string{}, "Namespaces to upgrade")
 	cmd.Flags().BoolVarP(&o.SkipCertManager, "skip-certmanager", "", false, "Skips certmanager installation")
 	cmd.Flags().StringArrayVarP(&o.Services, "services", "", []string{}, "Services to upgrdde")
+	cmd.Flags().BoolVarP(&o.SkipJxResourcesUpdate, "skip-jx-resources-update", "", false, "Skips the update of jx related resources such as webhook or Jenkins URL")
```
Comment (agentgonzo) [inline pkg/jx/cmd/upgrade_ingress.go]: We know this is jx, the flag could probably be `skip-resources-update` (and so could the variable names)

Comment (ccojocar) [reply pkg/jx/cmd/upgrade_ingress.go]: make sens

**Pull request merged**